[2024-09-12T02:21:29.649+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-12T02:21:29.690+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V6.third_task scheduled__2023-02-28T00:00:00+00:00 [queued]>
[2024-09-12T02:21:29.711+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V6.third_task scheduled__2023-02-28T00:00:00+00:00 [queued]>
[2024-09-12T02:21:29.715+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-12T02:21:29.759+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): third_task> on 2023-02-28 00:00:00+00:00
[2024-09-12T02:21:29.783+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=8185) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-12T02:21:29.792+0000] {standard_task_runner.py:72} INFO - Started process 8202 to run task
[2024-09-12T02:21:29.789+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V6', 'third_task', 'scheduled__2023-02-28T00:00:00+00:00', '--job-id', '20185', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp9blv9y1w']
[2024-09-12T02:21:29.794+0000] {standard_task_runner.py:105} INFO - Job 20185: Subtask third_task
[2024-09-12T02:21:29.913+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V6.third_task scheduled__2023-02-28T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-12T02:21:30.161+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='acap' AIRFLOW_CTX_DAG_ID='our_first_dag_V6' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2023-02-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-02-28T00:00:00+00:00'
[2024-09-12T02:21:30.168+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-12T02:21:30.208+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-12T02:21:30.210+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the third task!']
[2024-09-12T02:21:30.255+0000] {subprocess.py:86} INFO - Output:
[2024-09-12T02:21:30.258+0000] {subprocess.py:93} INFO - hey, this is the third task!
[2024-09-12T02:21:30.263+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-12T02:21:30.337+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-12T02:21:30.341+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V6, task_id=third_task, run_id=scheduled__2023-02-28T00:00:00+00:00, execution_date=20230228T000000, start_date=20240912T022129, end_date=20240912T022130
[2024-09-12T02:21:30.460+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-12T02:21:30.489+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
